cloud/k8s/deploy/kontain-deploy/base/cm-containerd-install.yaml

```diff
@@ -31,22 +31,29 @@ data:
     set -euo pipefail
 
     ROOT_MOUNT_DIR="${ROOT_MOUNT_DIR:-/root}"
+
+    echo logging the instance this is running on for debugging purposes
+    chroot ${ROOT_MOUNT_DIR} echo running on $(hostname)
```

Comment:
Debug?

And I completely miss the point of chroot here

Reply:
Agreed on this one.  Vestige - will remove in next edit.

Reply:
Actually, just remembered we need to identify which hostname of the worker node that we are running this instance of the daemon script on, so when the log is collected by a log collection agent or during debug, we know which node in the cluster it was run for.